### PR TITLE
Create LIT7071Qene.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Again, I was puzzled with the translation of the title, that makes no sense to me. Probably it is better to shorten the title, if we do not find a better and more meaningful translation.

### CODE REVIEW

File: new/LIT7071Qene.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55vb. Division into verses is indicated with ። .
+                    The end of the poem is indicated with ። ። .</note>
+                <ab>
+                    <l n="1">ለሰብአ፡ ማኅበር፡ ጸሪቀ፡ ማኅበሮሙ፡ ጳውሎስ፡ ድኅረ፡ ተቀስመ፡ በፄው፡ ወድኅረ፡ በእሳት፡ በሰለ።</l>
+                    <l n="2">ቅድመ፡ ገጸ፡ ኵሉ፡ በመጥባሕት፡ ተከፍለ።</l>
+                    <l n="3">ወኀብስት፡ ቤቶሙ፡ ጴጥሮስ፡ ዘርኢናሁ፡ ጥሉለ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7071Qene.xml:65]: ወኅብስተ፡ ?

File: new/LIT7071Qene.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7071Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለሰብአ፡ ማኅበር፡ ጸሪቀ፡ ማኅበሮሙ፡ ጳውሎስ፡ ድኅረ፡ ተቀስመ፡ በፄው፡ ወድኅረ፡ በእሳት፡ በሰለ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">La-sabʾa māḫbar ṣariqa māḫbaromu Ṗāwlos dǝḫra taqasma baḍew 
+                    wa-dǝḫra ba-ʾǝsāt basala. (Qǝne of the śǝllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">For the people of the monastery / part of their monastery / Paul / after it was made tasty in salt,
```
Comment (karljonaskarlsson) [inline new/LIT7071Qene.xml:11]: It seems to be connected to the martyrdoms of Peter and Paul. The first part, which must be a "synonym" for Paul, is what confuses me. Maybe:

"The communal coin of the people of the community, Paul, after being seasoned with salt and cooked on fire…"

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

File: new/LIT7071Qene.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7071Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለሰብአ፡ ማኅበር፡ ጸሪቀ፡ ማኅበሮሙ፡ ጳውሎስ፡ ድኅረ፡ ተቀስመ፡ በፄው፡ ወድኅረ፡ በእሳት፡ በሰለ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">La-sabʾa māḫbar ṣariqa māḫbaromu Ṗāwlos dǝḫra taqasma baḍew 
+                    wa-dǝḫra ba-ʾǝsāt basala. (Qǝne of the śǝllāse-type)</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7071Qene.xml:10]: ba-ḍew, no dot after "basala".

File: new/LIT7071Qene.xml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                <title xml:lang="en" corresp="#t1">The communal coin of the people of the community, Paul, after being seasoned with salt and
```
Comment (DenisNosnitsin1970) [inline new/LIT7071Qene.xml:11]: Yes, it is tricky. I think "ṣariq" is not only coin. It seems to mean smth like bread offering in the church,  of a special type. (Possibly small and thin, like a coin). I remember that accusative may mean also "like...", "in the shape of..." Maybe smth like this: For the people ot the community after Paul was seasoned with salt and cooked on fire, like bread offering of their community ...  Yes, there is an allusion to the martyrdom.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Review by karljonaskarlsson — APPROVED** [review]

File: new/LIT7071Qene.xml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7071Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለሰብአ፡ ማኅበር፡ ጸሪቀ፡ ማኅበሮሙ፡ ጳውሎስ፡ ድኅረ፡ ተቀስመ፡ በፄው፡ ወድኅረ፡ በእሳት፡ በሰለ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">La-sabʾa māḫbar ṣariqa māḫbaromu Ṗāwlos dǝḫra taqasma ba-ḍew 
+                    wa-dǝḫra ba-ʾǝsāt basala (Qǝne of the śǝllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">For the people of the community after Paul was seasoned with salt and cooked on fire, 
+                    like bread offering of their community …</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7071Qene.xml:12]: Can it be that there is an unnecessary space before ...?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7071Qene.xml (hunk 5 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7071Qene.xml:12]: There is definitely a space before the three dots. If you think, this is unnecessary, I will delete it. I do not know, what looks better in the app.

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**